packages/stark-core/src/common/error/index.ts

```diff
@@ -0,0 +1,4 @@
+"use strict";
```

Comment:
Now we don't need to add this "use strict", it is already taken care by Typescript compiler thanks to the "alwaysStrict" option enabled in the tsconfig.json ;)

Can you remove it from the rest of the files of this PR?